zendesk/data_source_zendesk_ticket_field_test.go

```diff
@@ -13,7 +14,7 @@ import (
 
 const systemFieldConfig = `
 data "zendesk_ticket_field" "assignee" {
```

Comment:
we should probably fix the example ticket fields as well. https://github.com/nukosuke/terraform-provider-zendesk/blob/master/examples/ticket_fields.tf#L58-L81

Reply:
i don't know if it needs to be @nukosuke or myself but someone should probably update the wiki as well. https://github.com/nukosuke/terraform-provider-zendesk/wiki/Data-Source:-zendesk_ticket_field

Reply:
I already updated the examples in the `terraform-v0.12` branch (which this PR targets). I would update the wiki as well, but I'm not sure I have permission.